test/checks/aria/required-children.js

```diff
@@ -10,97 +11,141 @@ describe('aria-required-children', function () {
 		}
 	};
 
+	function checkSetup (html, options, target) {
```

Comment:
This will have to be refactored once @WilcoFiers pull request gets merged - perhaps you should copy that change so that it will merge and eliminate refactoring?